virtual_ecosystem/models/abiotic/abiotic_model.py

```diff
@@ -54,6 +54,8 @@ class AbioticModel(
         ("air_temperature_ref", ("spatial",)),
         ("relative_humidity_ref", ("spatial",)),
         ("topofcanopy_radiation", ("spatial",)),
+        ("leaf_area_index", ("spatial",)),
+        ("layer_heights", ("spatial",)),
```

Comment:
In the final version, the `required_init_vars` are just a tuple, like `vars_updated` and the others since the axis information is contained in the variable description in the toml file.
```suggestion
        "air_temperature_ref",
        "relative_humidity_ref",
        "topofcanopy_radiation",
        "leaf_area_index",
        "layer_heights",
```
This change will make the validator fail, as it is expecting a different structure, but we will deal with that once this is merged, as it will be necessary to do some work harmonizing things.

Reply:
Yup - that sounds right. We're centralising the axis requirements, rather than allowing them to potentially clash between models, so all we need do in the model is provide the names of the variables needed.